test/e2e/app-dir/metadata-suspense/index.test.ts

```diff
@@ -10,13 +10,18 @@ describe('app dir - metadata dynamic routes suspense', () => {
     return
   }
 
-  it('should render metadata in head even root layout is wrapped with Suspense', async () => {
-    const $ = await next.render$('/')
+  it('should render metadata in head when root layout is wrapped with Suspense for bot requests', async () => {
```

Comment:
how come this test changed?

Reply:
What if metadata doesn't suspend? Wouldn't that end up in the head?

Reply:
Right, even it's pure static and your page static, the metadata can still be suspended and streamed to body. This is also what we want as it's not always be part of the shell. It's a normal suspenseful content now